series.go

```diff
@@ -5,20 +5,25 @@ import (
 	"net/http"
 )
 
+// AddSeries creates a new series in the database.
 func AddSeries(seriesId string) Request {
 	return Request{
 		Path:   fmt.Sprintf("/series/%s", seriesId),
 		Method: http.MethodPut,
 	}
 }
 
+// DeleteSeries deletes the series of the given seriesId from the database.
+//
+// Deleting a series will only delete assignment of items to it, not the items themselves!
```

Comment:
If you use `WithCascade` it will be removed, I think. Check it in documentation and add it here or in `WithCascade` options description